accname/name/comp_tooltip.html

```diff
@@ -11,11 +11,38 @@
 </head>
 <body>
 
-  <p>Tests the <a href="https://w3c.github.io/accname/#comp_tooltip">#comp_tooltip</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
+<p>Tests the <a href="https://w3c.github.io/accname/#comp_tooltip">#comp_tooltip</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<a href="#" title="label" data-expectedlabel="label" data-testname="link label from tooltip" class="ex"><img src="#" alt=""></a>
+<a href="#" title="label" data-expectedlabel="label" data-testname="link with img with tooltip label" class="ex"><img src="#" alt=""></a>
+<a href="#" title="label" data-expectedlabel="label" data-testname="link with text with tooltip label" class="ex">content</a>
+<div title="label" data-expectedlabel="label" data-testname="div with text with tooltip label" class="ex">content</div>
```

Comment:
This `div` test may be ambiguous since its role is undefined and should in theory be `""`, `generic`, or `none`, which don't support name from: `author` or `contents`. Adding `role="group"` should remove the ambiguity IIRC.

Reply:
Please file this test as-is, in a new, separate issue (on this repo or AccName) to make sure there isn't a spec ambiguity.

Reply:
> Please file this test as-is, in a new, separate issue (on this repo or AccName) to make sure there isn't a spec ambiguity.

Got it, filed https://github.com/w3c/accname/issues/202

Reply:
> This `div` test may be ambiguous since its role is undefined and should in theory be `""`, `generic`, or `none`, which don't support name from: `author` or `contents`. Adding `role="group"` should remove the ambiguity IIRC.

Done in https://github.com/web-platform-tests/wpt/pull/42093/commits/73e4577022000d4a48a74c042a4925837cd6cbd2